moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -90,10 +90,8 @@ export const ProjectsListViewFiltersConf = {
         "string_contains_case_insensitive",
         "string_begins_with_case_insensitive",
         "string_ends_with_case_insensitive",
-        // Because of the way the SQL view concatanates the team members and roles
-        // " :" is always present even if there are no team members present
-        "string_is_null_special_case",
-        "string_is_not_null_special_case",
+        "string_is_null",
+        "string_is_not_null",
```

Comment:
we don't need to handle a special case where no team members were showing up as " : "